react-beers-js/src/components/NewBeer.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+import Header from "./Header.js";
+
+class NewBeer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      tagline: "",
+      description: "",
+      first_brewed: "",
+      brewers_tips: "",
+      attenuation_level: 0,
+      contributed_by: ""
+    };
+  }
+
+  handleFormSubmit = event => {
+    event.preventDefault();
+    const name = this.state.name;
+    const tagline = this.state.tagline;
+    const description = this.state.description;
+    const first_brewed = this.state.first_brewed;
+    const brewers_tips = this.state.brewers_tips;
+    const attenuation_level = this.state.attenuation_level;
+    const contributed_by = this.state.contributed_by;
+
+    axios
+      .post("https://ironbeer-api.herokuapp.com/beers/new", {
+        name,
+        tagline,
+        description,
+        first_brewed,
+        brewers_tips,
+        attenuation_level,
+        contributed_by
+      })
```

Comment:
Comme ton state ne contient que les informations à envoyer à l'API et que les propriétés de ton state ont les mêmes noms que ce que l'API attends, tu aurais pu passer this.state en deuxième argumant de la fonction post